sqlmesh/dbt/model.py

```diff
@@ -269,9 +272,11 @@ def to_sqlmesh(self, context: DbtContext) -> Model:
                 d.parse_one(c, dialect=dialect).name for c in self.cluster_by
             ]
 
-        for field in ["cron", "forward_only", "disable_restatement"]:
+        for field in ["cron", "interval_unit", "forward_only", "disable_restatement"]:
             field_val = getattr(self, field, None) or self.meta.get(field, None)
             if field_val:
+                if field == "interval_unit":
+                    field_val = IntervalUnit(field_val.lower())
```

Comment:
This will fail with an obscure error message if the value is invalid. Can we instead just pass it on and let sqlmesh do the validation?